posts/crowdsales-and-multiple-inheritance.md

```diff
@@ -0,0 +1,327 @@
+On 2017 we saw a mind-blowing number of crowdsales. They have proved to be a
```

Comment:
Perhaps you want specifically refer to crowdsales running in the ethereum blockchain or ICOs?

Reply:
Also on this line, I think its "proven" instead of "proved".